engine/dlib/src/test/test_buffer.cpp

```diff
@@ -217,6 +217,126 @@ TEST_F(BufferTest, ValidateBuffer)
     dmBuffer::Destroy(buffer);
 }
 
+TEST_F(BufferTest, MetaData)
```

Comment:
Try not to put all the tests in the same test function.
It's a lot easier to read and reason about if the tests are smaller.

Reply:
ok